StripePaymentsUI/StripePaymentsUI/Source/UI Components/CustomerContext/STPEphemeralKeyManager.swift

```diff
@@ -65,7 +65,7 @@ class STPEphemeralKeyManager: NSObject, STPEphemeralKeyManagerProtocol {
         )
     }
 
-    @objc dynamic func getOrCreateKey(_ completion: @escaping STPEphemeralKeyCompletionBlock) {
+    @objc @_spi(STP) public dynamic func getOrCreateKey(_ completion: @escaping STPEphemeralKeyCompletionBlock) {
```

Comment:
Curious as to why this needs to be @_spi(STP) public -- the caller is `_stpspmsbeta_STPCustomerContext` which also lives in the same module -- i guess i can see it being 'public' in the even that users implement their own STPCustomerContext, but not sure i fi understand why `@_spi(STP) public` is needed.

Reply:
Stripe.framework still uses it for Issuing, which we haven't split into a separate SDK yet.

Reply:
Chatted offline about this  -- was looking at some outdated code. my bad!